app/models/location.py

```diff
@@ -29,20 +38,18 @@ def create_zip_code(zip_code):
         """
         result = ZIPCode.get_by_zip_code(zip_code)
         if result is None:
-            result = ZIPCode(zip_code)
+            result = ZIPCode(zip_code=zip_code)
             db.session.add(result)
             db.session.commit()
         return result
 
     @staticmethod
-    def generate_fake(count=10):
+    def generate_fake():
         """Generate count fake ZIPCodes for testing."""
-        from faker import Faker
+        zippy = ['19104', '01810', '02420', '75205', '94305', '47906', '60521']
 
-        fake = Faker()
-
-        for i in range(count):
-            ZIPCode.create_zip_code(fake.zipcode())
+        for zip in zippy:
```

Comment:
don't use `zip` as a variable because it's a python function and you're probably overloading it, `zip_code` should do.